### ISSUE TITLE
backup to cloud wants to disconnect my account

### ISSUE THREAD

**Issue by phpmoli** [AndreAle94/moneywallet#258]
Hello,

I switched my phone from Xiaomi's Android One to Lineage OS, since, when i _initiate_ a backup to an _already_ connected dropbox account, a popup appears asking "are you sure you want to disconnect your dropbox account" ? And if i cancel/no this question the backup process is saying a json error along the lines of token expired. This happens even after disconnecting my dropbox and then _reconnecting_ my dropbox again, and in a new flow(!) initiating the backup process _seconds_ after getting the brand new token (in the first backup flow). The only way i can back up to dropbox if i reconnect/get the token every single time i want to back up.

So i thought, i am gonna use my google account instead of my dropbox. Nope, same thing happens, when _initiating_ the token getting-process with google, _in this flow_, after selecting my google account in the webview(?), the same "are you sure you want to disconnect your google account" dialog pops up, _while_ i am trying to connect to my account for the _first time_ and never ever in my life was connected to my google account in moneywallet before (so there is nothing to disconnect).

This is happening on a brand new reinstall and database import too.

If you need more data, please tell.

moneywallet v4.0.5.10
lineageos v20
android v13
google play services v23.05.14
webview v108.0.5359.128

**Comment by EdMaxPrime** [comment]
Are you able to create the full backup file in your filesystem and then upload it to a cloud service manually?

**Comment by phpmoli** [comment]
Yes, that's how i use it nowadays :( That backup is valid, tested it.
